### ISSUE TITLE
[🐞] ITMS-90809: Deprecated API Usage , PayPal Mobile iOS SDK

### ISSUE THREAD

**Issue by Cniklasson** [paypal/paypalcheckout-ios#177]
### 🐞 Describe the Bug

Dear Developer,

We identified one or more issues with a recent delivery for your app, "Beliri" 1.0.1 (12). Please correct the following issues, then upload again.

ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview).

### 🔬 Minimal Reproduction
How to move forward from here?

### 😕 Actual Behavior
Developer have just integrated PayPal Mobile iOS SDK and was trying to upload the build to Test Flight and recived this error. 

### 🤔 Expected Behavior
To run smoothly.

### 🌍 Environment

- iOS version: - 17.2.1
- SDK version (`Checkout.version`): - Latest?
![IMG_4686](https://github.com/paypal/paypalcheckout-ios/assets/148633934/6a26b974-5171-46a6-869f-4d5f55ad00e7)

### ➕ Additional Context
Add any other context about the problem here. Screenshots or videos that show the issue are very helpful.

**Comment by Cniklasson** [comment]
PayPal Mobile SDK
Version : 2.18.1

**Comment by Cniklasson** [comment]
@jonathajones Any idea?

**Comment by mariolopez-pypl** [comment]
@Cniklasson Hi there, thank you for reaching out. We currently do not use UIWebview and we do use WKWebView. We do not have a version 2.18.1. We have only released up to 1.2.0. Can you double check and make sure the library you are using is this one and that the version is 1.2.0?

**Comment by Cniklasson** [comment]
> @Cniklasson Hi there, thank you for reaching out. We currently do not use UIWebview and we do use WKWebView. We do not have a version 2.18.1. We have only released up to 1.2.0. Can you double check and make sure the library you are using is this one and that the version is 1.2.0?

Many thanks for the reply. It was a missunderstanding. The version developer is using is 1.0.0

” We have followed this url for integration https://developer.paypal.com/docs/checkout/advanced/ios/ 

Native payments

Version 1.0.0”

**Comment by mariolopez-pypl** [comment]
@Cniklasson Thank you for the clarification. On our end in 1.0.0 we do not use UIWebViews as well, can you verify on your end that you are not using a UIWebView in your own app? And can you make sure you are using the latest 1.2.0 version?

**Comment by Cniklasson** [comment]
> @Cniklasson Thank you for the clarification. On our end in 1.0.0 we do not use UIWebViews as well, can you verify on your end that you are not using a UIWebView in your own app? And can you make sure you are using the latest 1.2.0 version?

Hello,
I wrote to developer 

”https://github.com/paypal/paypal-ios/releases

Bump PayPalCheckout to 1.2.0
Can you update it
And upload new build to test flight
And we go from there”

They wrote me back

 ”Hi Christian 

I have a concern here. As previously discussed we are not using paypal checkout, we are using paypal native payments now. Please check with paypal support once. We need to resolve paypal native payment issue.”

its still the link above i have shared?

Many thanks for the support

**Comment by Cniklasson** [comment]
@mariolopez-pypl 
@jonathajones 


I recived this email from PayPal support.

Just a reminder, we are using PayPal Native Payments. 

How do i move from here?

happy for all help

”Hi Christian Niklasson, I hope you're doing well.

Thank you for your response.

The PayPalCheckout 1.2.0 version is not the latest version for PayPal SDK.

The latest version is PayPalCheckout 4.x.x.

To ensure that you have the correct and latest update from the start, please follow these steps:
1. Remove any previous versions of the PayPal SDK from your project.
2. It is recommended to use a package manager or dependency management tool like Composer or npm to install the PayPalCheckout SDK. This will ensure that you get the latest version and also simplify the installation process.

By using the package manager, you will automatically get the latest version of the PayPalCheckout SDK, allowing you to benefit from the latest features, bug fixes, and security improvements.

I hope this was helpful. Let me know if you need any further information on this. Thank you for your patience and understanding.

Regards,
Sanchit Aggarwal
Merchant Technical Support
PayPal”

**Comment by mariolopez-pypl** [comment]
@Cniklasson 

What the merchant technical support specialist is referring is an NPM package which is (node package manager). Which is used for Node.js projects (https://www.npmjs.com/package/paypal-checkout). For iOS though we use a different library manage, Cocoapods and only support up to 1.2.0. So to use this project 1.2.0 is the recommend version.

**Comment by mariolopez-pypl** [comment]
@Cniklasson

This is specific to PayPalNativePayments, posting the question in this [repo](https://github.com/paypal/paypal-ios). This we are in right now is specifically for PayPalCheckout. Thank you

**Comment by Cniklasson** [comment]
> @Cniklasson
> 
> This is specific to PayPalNativePayments, posting the question in this [repo](https://github.com/paypal/paypal-ios). This we are in right now is specifically for PayPalCheckout. Thank you

Thank you so much @mariolopez-pypl 

So my questions should be made here? https://github.com/paypal/paypal-ios

As we are following this link 
https://developer.paypal.com/docs/checkout/advanced/ios/

Then Native Payments.

Is that correct?

**Comment by mariolopez-pypl** [comment]
Yes that is correct, questions should be made there. That repo you linked (https://github.com/paypal/paypal-ios), is the repo that is talked about in the doc link you sent.
